packages/wonder-blocks-tooltip/components/tooltip-anchor.js

```diff
@@ -6,21 +6,23 @@
 import * as React from "react";
 import * as ReactDOM from "react-dom";
 
-import TooltipPortalMounter from "./tooltip-portal-mounter.js";
+import {Text as WBText} from "@khanacademy/wonder-blocks-core";
```

Comment:
Why WBText?? This seems really awkward.

Reply:
I'll add a comment but it's b/c there's a React.Text type... I guess I could check for that instead of `Text` and then this could be simply `Text`.

Reply:
I had this issue somewhere too. I don't think `React.Text` worked for me, so `WBText` was the way I went. I recommend doing the same.